mgmtfn/k8splugin/README.md

```diff
@@ -0,0 +1,311 @@
+# Contiv Networking for Kubernetes
+
+Contiv is integrated with Kubernetes via a CNI plugin. This integration enables you
+to set up and manage inter-pod connectivity in a Kubernetes cluster using Contiv 
+policy model.
+
+## Getting Started
+
```

Comment:
> Making/building binaries will go away soon -- there is a follow up PR coming.
> Checking out repos -- the ansible repo will need to be checked out, unless we want to provide a .tgz.

+1 
Let's track it with an issue if you are going to work on it later; that way someone else can also pick it up.

> I did not quite follow this one:
"ansible work to install things (those all should be part of vbox image, for Vagrant)"

> The ansible repo is there and the packages are all pre-installed in the vbox.

Then don't bother with my comment; I was suggesting that ansible should not install packages.

> In terms of the steps, are you suggesting there should be 4 steps instead of the 7 that I have? I think we will automatically get there when the pull repo/build binaries part goes away.

That would be awesome!